umbraco-cloud/set-up/project-settings/manage-hostnames/README.md

```diff
@@ -26,7 +26,11 @@ Under **Settings** in the Umbraco Cloud Portal, you'll find **Hostnames**. This
 
 Ensure that the hostname you are binding to your Umbraco Cloud environment has a DNS entry that resolves to the Umbraco Cloud service.
 
-We recommend setting a CNAME record for your hostname. You can also use A & AAAA records for the root of your domain.
+A & AAAA records should be created for the root of your domain.
+
+For any subdomain, we recommended setting a CNAME to either the root of your domain or to dns.umbraco.io.
```

Comment:
I have updated this part a bit to make it more clear when to use what. I have also removed the part where it says to add cname at the root to not confuse others and as in current docs we recommend to point to dns.umbraco.io and not another domain that points to this. When this is confirmed internally the docs will be updated as well. I have also removed mention of subdomain from CNAME as it is already in the info note. 
```suggestion
- A & AAAA records: they are usually used for the Apex domain (without "`www`" in the URL), and needs to be created at the root of your domain.

- CNAME: usually used for domains with "`www`" in the URL, recommended to use if possible, as the record is not changed as often as A & AAAA records are. When setting up a CNAME it needs to point to `dns.umbraco.io`.
```